wireguard-server/entrypoint.sh

```diff
@@ -37,6 +37,6 @@ trap 'cleanup' SIGTERM
 
 wgrest_auth_token=$(cat /run/secrets/wgrest_auth_token)
```

Comment:
Probably more of a nit for the challenge, but I would consider generating a secure random wgrest auth token for every "docker compose" (and passing it to the wirescale server as a parameter) instead of having a hardcoded credential.

Reply:
I agree - but part of the reason I hardcoded it here [was actually to make it obvious that this wasn't supposed to be used for a production system](https://github.com/runesoerensen/wirescale/commit/8877dacdbe5d55185b45b2f70308695185decd33) (where it's unlikely that the secret would ever be committed to version control or, I suppose, be generated on every deployment).

Happy to change it to generate a secret on every compose, but hope that makes sense?